synapse/cortex.py

```diff
@@ -1888,6 +1888,16 @@ def _getTagPrune(self, tagname):
 
         return tuple(prune)
 
+    async def getTagNorm(self, tagname):
```

Comment:
Since these aren't async operations, this API could squeek out a tiny extra bit of perf by not being async either.